stufftodo.txt

```diff
@@ -0,0 +1,3 @@
+1. are we going to use Systick for both motor moves
```

Comment:
It makes sense to me to use SysTick for both motors. There is a change to exactly how that works with the motor drivers; they have a two-pin interface; A 'step' pin and a 'direction' pin. We could use a state machine like we used before, excluding some of the steps specific to the timer project. One for each motor would allow them to move independently, and we would just make sure they have both reached the 'done' state before moving on to what ever is next.